.gitattributes

```diff
@@ -0,0 +1,7 @@
+# mocked files
+mock_*.go binary
+*_mock.go binary
```

Comment:
Mainly to make the diff view in the code reviews a little smaller. Phabricator suppresses   auto-loading files marked `binary`.  GH does the same for files marked `linguist-generated`. [reference]

[reference]: https://robots.thoughtbot.com/github-diff-supression